bind/flag.go

```diff
@@ -105,6 +105,29 @@ func HTTPProxyConfig(fs *pflag.FlagSet, cfg *forwarder.HTTPProxyConfig, lcfg *lo
 			"By default, requests to localhost are denied. ")
 }
 
+func MITMConfig(fs *pflag.FlagSet, mitm *bool, cfg *forwarder.MITMConfig) {
+	fs.BoolVar(mitm, "mitm", *mitm, ""+
+		"Enable Man-in-the-Middle (MITM) mode. "+
+		"It only works with HTTPS requests, HTTP/2 is not supported. "+
+		"MITM is enabled by default when the --mitm-cacert-file flag is set. "+
+		"If the CA certificate is not provided MITM uses a generated CA certificate. "+
+		"The CA certificate used can be retrieved from the API server .")
+
+	fs.StringVar(&cfg.CACertFile, "mitm-cacert-file", cfg.CACertFile, "<path or base64>"+
```

Comment:
Cert and Key flags have `-file` suffix, which we previously removed from other cert flags. What's the reason behind this? Do we want to keep SC 4.X naming? 
IMO we must have the same naming convention for all cert related flags.

Reply:
We did not yet remove them. There is a separate ticket for it https://github.com/saucelabs/forwarder/issues/337.